app/components/common/form/SaveButton.tsx

```diff
@@ -0,0 +1,18 @@
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+
+export function FormSaveButton({ onSave, disabled }: { onSave: () => void; disabled: boolean }) {
+  return (
+    <Box display="flex" justifyContent="flex-end">
+      <Button
+        onClick={onSave}
+        variant="contained"
+        size="small"
+        disabled={disabled}
+        sx={{ width: '30%', backgroundColor: 'secondary.main' }}
+      >
+        Speichern
```

Comment:
add translation entries for this please:

- [ ] en: "components_profilePage_form_updateUserForm_save": "Save"
- [ ] de: "components_profilePage_form_updateUserForm_save": "Speichern"